templates/tpl.BigBlueButtonClient.html

```diff
@@ -16,12 +16,7 @@
 				$('#classNotStarted').show();
 			}
                         
-                        var isMeetingRecorded={isMeetingRecorded};
```

Comment:
Why did u remove this?
This is the hint if the current running meeting is recorded or not.

Reply:
Oh yes i forgot it the current meeting recording hint should be as it was before.By mistakenly in testing i removed it.